frameworks/helloworld/tests/test_multiservice_dynamic.py

```diff
@@ -111,7 +111,6 @@ def test_add_deploy_restart_remove():
 
 
 @pytest.mark.sanity
-@pytest.mark.smoke
```

Comment:
Will this test be re-tagged with `smoke` in the context of DCOS-38379?

Reply:
Yes. Let me add a comment to the issue.